draft-ietf-quic-multipath.md

```diff
@@ -957,8 +964,8 @@ at any time during the connection. As such a sole change of the Connection
 ID without any change in the address does not indicate a path change and
 the endpoint can keep the same congestion control and RTT measurement state.
 
-While endpoints assign a connection ID to a specific sending 4-tuple,
-networks events such as NAT rebinding may make the packet's receiver
+While endpoints send a packet on a specific path with the associated
+4-tuple, networks events such as NAT rebinding may make the packet's receiver
```

Comment:
I proposed to remove this text in PR #389 as there is nothing special here anymore. It just describing RFC9000 behavior.

Reply:
Yes, we may have a merge conflict with PR #389. But if we delete it now, we will have a different conflict. What do you suggest? Keep the old text for now, knowing that it will be removed?

Reply:
OK, reverting that particular change to avoid a merge conflict with #389.